README.rst

```diff
@@ -69,22 +76,20 @@ What are you looking for?
 =========================
 
 * `Installation Instructions <https://renate.readthedocs.io/en/latest/getting_started/install.html>`_
-
     .. code-block:: bash
 
       pip install renate
 
-* Examples
-    We provide examples to train your model
-    `locally <https://renate.readthedocs.io/en/latest/examples/train_mlp_locally.html>`_
-    or `using Amazon SageMaker <https://renate.readthedocs.io/en/latest/examples/train_classifier_sagemaker.html>`_.
-* `Documentation <https://renate.readthedocs.io>`_
-* `Supported Algorithms <https://renate.readthedocs.io/en/latest/getting_started/supported_algorithms.html>`_
-* `Experimenting with Renate <https://renate.readthedocs.io/en/latest/benchmarking/index.html>`_
+* Examples:
```

Comment:
the "what are you looking for" became redundant after the introduction of the quicklinks. drop it altogether?

Reply:
I think it's still valuable to have the links at the bottom. I would like to keep them.